opentracing-cdi/src/main/java/io/opentracing/contrib/cdi/internal/SpanContextProducer.java

```diff
@@ -35,13 +36,13 @@ public SpanContext produceSpanContext() {
     }
 
     @Produces
-    public ActiveSpan produceActiveSpan(InjectionPoint ip) {
-        ActiveSpan activeSpan = tracer.activeSpan();
-        if (null == activeSpan) {
+    public Span produceActiveSpan(InjectionPoint ip) {
```

Comment:
Think it may be better to have `Scope` as the injected entity - so then equivalent to the active span from before.

Reply:
I added another producer for the `Scope`. This way, we don't break existing code that expect to get a `Span` injected. Code that expects an `ActiveSpan` injected will still fail, though.

Reply:
👍 Good compromise.